modules/hitlnext/src/views/full/Api.ts

```diff
@@ -93,6 +94,10 @@ export const Api = (bp: { axios: AxiosInstance }): ApiType => {
         .post(`/handoffs/${id}/resolve`, null, config)
         .then(res => res.data)
         .then(data => castHandoff(data)),
+    deleteMessages: async (id, userId, channel) =>
```

Comment:
can you just rename this function for (deleteMessagesInChannelWeb) then you'll be all set !